coralproject/talk
# [CORL-2375] Use worker threads to process regex to keep regex operations async friendly

NOTE: this PR is branched off of CORL-2300's PR, merge that first and retarget this to develop before merging this PR.

## What does this PR do?

Create an internal wordlist service that uses worker threads

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

- set banned and suspect words in the config
- create comments with commenter account using those words
- see them get detected in mod queues and/or rejected from stream
 
## How do we deploy this PR?

Nothing special here.
